Subject: Always use PERLRUNINST when building perl modules.
Bug-Debian: http://bugs.debian.org/357264
Bug: http://rt.cpan.org/Public/Bug/Display.html?id=17224

Revert part of upstream change 24524 to always use PERLRUNINST when
building perl modules:  Some PDL demos expect blib to be implicitly
searched.


---
 lib/ExtUtils/MM_Unix.pm |    5 +----
 1 files changed, 1 insertions(+), 4 deletions(-)

diff --git a/lib/ExtUtils/MM_Unix.pm b/lib/ExtUtils/MM_Unix.pm
index ad28b22..1f6b2ef 100644
--- a/lib/ExtUtils/MM_Unix.pm
+++ b/lib/ExtUtils/MM_Unix.pm
@@ -3031,14 +3031,11 @@ sub processPL {
 	    # pm_to_blib depends on then it can't depend on pm_to_blib
 	    # else we have a dependency loop.
 	    my $pm_dep;
-	    my $perlrun;
 	    if( defined $self->{PM}{$target} ) {
 		$pm_dep  = '';
-		$perlrun = 'PERLRUN';
 	    }
 	    else {
 		$pm_dep  = 'pm_to_blib';
-		$perlrun = 'PERLRUNINST';
 	    }
 
             $m .= <<MAKE_FRAG;
@@ -3047,7 +3044,7 @@ all :: $target
 	\$(NOECHO) \$(NOOP)
 
 $target :: $plfile $pm_dep
-	\$($perlrun) $plfile $target
+	\$(PERLRUNINST) $plfile $target
 MAKE_FRAG
 
 	}
-- 
tg: (daf8b46..) fixes/processPL (depends on: upstream)
